src/main/scala/hellotest/Main.scala

```diff
@@ -62,10 +62,10 @@ object Main:
     }
 
     // 4. discard sequences of non-valid length
-    val validLengthSequences = stringSequences // DO:
+    val validLengthSequences = stringSequences.filter(seq => seq.length >= args.minLength)
```

Comment:
Also, I still recommend creating a separate function in the `FunctionalWordCloud.scala` file and unit test for this, so that you won't need to test this manually

Reply:
Done